lightning/src/routing/router.rs

```diff
@@ -607,19 +608,25 @@ fn compute_fees(amount_msat: u64, channel_fees: RoutingFees) -> Option<u64> {
 /// [`Event::PaymentPathFailed`]: crate::util::events::Event::PaymentPathFailed
 pub fn find_route<L: Deref, S: Score>(
 	our_node_pubkey: &PublicKey, route_params: &RouteParameters, network: &NetworkGraph,
-	first_hops: Option<&[&ChannelDetails]>, logger: L, scorer: &S
-) -> Result<Route, LightningError>
+	first_hops: Option<&[&ChannelDetails]>, logger: L, scorer: &S, random_seed_bytes: &[u8; 32]
+	) -> Result<Route, LightningError>
 where L::Target: Logger {
-	get_route(
+	match get_route(
 		our_node_pubkey, &route_params.payment_params, network, first_hops, route_params.final_value_msat,
-		route_params.final_cltv_expiry_delta, logger, scorer
-	)
+		route_params.final_cltv_expiry_delta, logger, scorer, random_seed_bytes
+		) {
+		Ok(mut route) => { 
```

Comment:
nit: here and in a few other places in this PR you introduce whitespace at EOL. A local `git show` should highlight these for you.

Reply:
Fixed with https://github.com/lightningdevkit/rust-lightning/commit/386a0b553130173f576ecc89424fcd6791a41dcc.